### PR TITLE
Fixes #15715: cli needs a rudder server health command

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/15715

### CODE REVIEW

File: share/commands/server-health [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+WAIT="no"
+TIMEOUT=60
+
+while getopts "wn" opt; do
+  case $opt in
+    w)
+      WAIT="yes"
+      ;;
+    n)
+      NRPE="no"
```
Comment (Fdall) [inline share/commands/server-health]: yes

File: share/commands/server-health [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  elif [ "${WAIT}" = "no" ]
+  then
+    echo "Server API Response is KO."
+    exit 2
+  elif [ $i -eq 60 ]
+  then
+    echo "Server Response is not KO after 60 seconds..."
+    exit 2
+  fi
+  sleep 2
```
Comment (Fdall) [inline share/commands/server-health]: 1

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**